src/website/map_object_info.cc

```diff
@@ -101,11 +101,16 @@ void write_buildings(const TribeDescr& tribe, EditorGameBase& egbase, FileSystem
 				}
 			}
 			// Produces workers
-			if (productionsite->output_worker_types().size() > 0) {
+			if (productionsite->output_worker_types().size() > 0
+				|| productionsite->type() == Widelands::MapObjectType::TRAININGSITE) {
```

Comment:
What is the purpose of this line above and the addition below?

Reply:
The training sites used to have "soldier" in their output table, but they don't technically produce soldiers. So, I'm adding a manual remedy for the website encyclopedia here.